source/educators/references/advanced_features/managing_teams_via_csv.rst

```diff
@@ -41,7 +41,7 @@ For example, a course with 2 team-sets (``discussion-teams`` and ``case-studies`
     user, mode, discussion-teams, case-studies
 
 Each additional row is a **user row** and has a ``user-identifier``
-(which, in priority order, is the ``student-key``, ``edx-username``, or ``edx-email``),
+(which, in priority order, is the ``student-key``, ``instance-username``, or ``institution-email``),
```

Comment:
'institution-email' makes me think this needs to be an .edu address or some other official email. Just change these to 'username' and 'email'